src/services/smart-fields-values-injector.js

```diff
@@ -68,9 +76,10 @@ function SmartFieldsValuesInjector(record, modelName, fieldsPerModel, depth = 0)
 
   this.perform = () =>
     P.each(schema.fields, (field) => {
-      if (!record[field.field]) {
+      if (record
+          && !Object.prototype.hasOwnProperty.call(record, field.field)) {
```

Comment:
This is the real fix.
You can follow this [discussion](https://github.com/ForestAdmin/forest-express/pull/470/files#r466433245) for more details.

Reply:
👍

Reply:
> I've tested it yet, but by reading this line I'm guessing that magic accessors are not considered as own properties. Correct ?

absolutely right sir!